src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should be more informative. Indicate that a `Truck` has stopped working, for instance: 'Truck has stopped its work.'